src/codegen.cpp

```diff
@@ -9645,8 +9645,8 @@ void jl_compile_workqueue(
         auto proto = it.second;
         params.workqueue.pop_back();
         // try to emit code for this item from the workqueue
-        assert(jl_atomic_load_relaxed(&codeinst->min_world) <= params.world &&
-               jl_atomic_load_relaxed(&codeinst->max_world) >= params.world &&
+        assert(jl_atomic_load_relaxed(&codeinst->min_world) <= params.min_world &&
+               jl_atomic_load_relaxed(&codeinst->max_world) >= params.max_world &&
```

Comment:
@Keno I believe this is correct, but probably worth double-checking.

Reply:
LGTM